processing/requirements.in

```diff
@@ -8,5 +8,5 @@ psycopg2-binary
 python-dateutil
 python-decouple
 requests
-scrapy
```

Comment:
**suggestion**: I used [pip-compile](https://github.com/jazzband/pip-tools) to generate `requirements.txt` from this `requirements.in` file, so we don't need to specify Scrapy version here.